### ISSUE TITLE
In v0.7, error requiring Polyhedra from LazySets

### ISSUE THREAD

**Issue by mforets** [JuliaReach/LazySets.jl#825]
```julia
julia> using LazySets
[ Info: Precompiling LazySets [b4f0291d-fe17-52bc-9479-3d1a343d9043]
WARNING: Method definition setrounding(Type{T<:Union{Float32, Float64}}, Base.Rounding.RoundingMode{T} where T) where {T<:Union{Float32, Float64}} in module Base at deprecat
ed.jl:1699 overwritten in module SetRounding at /Users/forets/.julia/packages/SetRounding/MjtxK/src/SetRounding.jl:47.


julia> using Polyhedra
┌ Warning: Package LazySets does not have CDDLib in its dependencies:
│ - If you have LazySets checked out for development and have
│   added CDDLib as a dependency but haven't updated your primary
│   environment's manifest file, try `Pkg.resolve()`.
│ - Otherwise you may need to report an issue with LazySets
└ Loading CDDLib into LazySets from project dependency, future warnings for LazySets are suppressed.
```

**Comment by mforets** [comment]
my expectation was that `CDDLib` was "transparent" to the user: upon doing `using Polyhedra`, the default backend `CDDLib` would be loaded immediately and be ready. this strategy worked in 0.6.

as it turns out, in the new `Requires` this is not available (unless im missing sth), and the user has also to do `using CDDLib`. then the order in which these packages are loaded matters because we have some initialization code in `__init__()`.

**Comment by schillic** [comment]
We could write a message "please load `CDDLib` in advance" if it is not loaded. I think it is better to explain the problem so a user who is interested can avoid the problem in the future.

**Comment by schillic** [comment]
This was resolved with #831.
